### PR TITLE
Optimize findAllFilesByExtension (less memory usage)

### PR DESCRIPTION
**Pull request by Tutul-**
- Use Array instead of ArrayList  (less memory usage)
- Can be faster when EXCLUDES_FILES is bigger, now it's a little bit slower (~0.030s)
BONUS : change .gitignore for Eclipse stuff and *.class

### CODE REVIEW

**Comment by qparis** [comment]
Ok to test

**Comment by qparis** [comment]
Test this please

**Comment by qparis** [comment]
Ok to test

File: src/main/java/com/playonlinux/wine/WinePrefix.java [hunk 1]
```diff
@@ -41,6 +44,7 @@
     private static final String EXECUTABLE_EXTENSION = "exe";
     private static final String DRIVE_C = "drive_c";
 
+    private static String[] EXCLUDES_FILES = new String[1]; // One element for now
```
Comment (qparis) [inline src/main/java/com/playonlinux/wine/WinePrefix.java]: It is better to directly set here the list of exclusion:
private static String[] EXCLUDES_FILES = new String[] { "iexplore.exe", "notepad.exe" }, etc...

**Comment by Tutul-** [comment]
Done

**Comment by qparis** [comment]
Ok to test

**Pull request merged**